Objective-C/CBLReplicator.mm

```diff
@@ -148,11 +129,6 @@ - (NSString*) description {
     return _desc;
 }
 
-- (void) clearRepl {
-    c4repl_free(_repl);
-    _repl = nullptr;
-}
-
```

Comment:
You can still tell that it has been closed/freed?

Reply:
i am assuming, we are not deleting the c4replicator instance, and it should be handled by lite core.

Reply:
👍